src/hotspot/share/ci/ciEnv.cpp

```diff
@@ -1773,12 +1773,7 @@ void ciEnv::dump_inline_data(int compile_id) {
 
 bool ciEnv::is_multifield_scalarized(BasicType bt, int vec_length) {
 #if COMPILER2
-  CompilerThread* ct = CompilerThread::current();
```

Comment:
Could you please change `#if COMPILER2` to `#ifdef COMPILER2` ? This makes the jdk build error with `--configure-args --with-jvm-features=-compiler2`